package.json

```diff
@@ -178,7 +178,7 @@
     "@nx/react": "19.5.3",
     "@nx/storybook": "20.7.1",
     "@nx/workspace": "20.6.2",
-    "@opentelemetry/semantic-conventions": "1.30.0",
+    "@opentelemetry/semantic-conventions": "^1.30.0",
```

Comment:
@nshirley I just reverted this pin, which should now resolve to 1.32.0 as you have in your PR.  Let's see if it passes tests!  (Seemed like it might locally.)

Reply:
Looks like it passes 🎉